src/pages/components/timeline.mdx

```diff
@@ -136,16 +148,25 @@ import COMPONENTS from '../../examples/accordions/components.ts';
 
 <Component components={COMPONENTS} componentName="timeline.content" />
 
+The Content component renders the provided content with a separator and an icon for the
```

Comment:
re: `separator` I'd like to steer away from styling-related descriptions, if possible. Also, isn't the `icon` part of the rendered `Timeline.Item` (vs `Timeline.Content`)?